students/urtnowski_daniel/lesson_12_debugging/verbose_output.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+"""
+verbose_output.py: a practice project: "Verbose output" from:
+Lesson 12 - Debugging
+
+The program is based on regex_search.py with configurable logger added.
+
+Usage: verbose_output.py [-h] -r REGEX [-p PATH] [-V {0,1,2}]
+"""
+
+__author__ = "Daniel Urtnowski"
+__version__ = "0.1"
+
+
+import re
+import os
+import sys
+import argparse
+import enum
+import datetime
+
+
+class Logger:
```

Comment:
Może polecenie nie było zbyt precyzyjne, ale to zadanie zostałoby również zaliczone gdybyś użył mechanizmu logowania z modułu `logging`, o którym mowa jest w materiałach teoretycznych do tej lekcji: https://automatetheboringstuff.com/chapter10/

Jeżeli nie znasz tego modułu, to zachęcam do zapoznania się z nim. Zadanie zaliczam, bo Twoja implementacja również spełnia wymagania.